modules/compress/src/test/java/org/apache/ignite/internal/processors/compress/CompressionTestSuite.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ignite.internal.processors.compress;
+
+import junit.framework.TestSuite;
+
+/**
+ */
+public class CompressionTestSuite extends TestSuite {
```

Comment:
We need to add more suites the same way Direct IO sutes are added with one default compression algorithm used to make sure all recovery mechanics works well. Once we introduce nightly builds, we may consider adding all compression algorithms to the nightly suite.

Reply:
Agree. Please provide the list of tests to be added.